apps/prairielearn/src/sync/course-db.ts

```diff
@@ -181,263 +191,232 @@ const UUID_REGEX = /[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-
 const FILE_UUID_REGEX =
   /"uuid":\s*"([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12})"/g;
 
-/**
- * @template T
- * @typedef {import('./infofile').InfoFile<T>} InfoFile<T>
- */
-
-/**
- * @typedef {Object} CourseOptions
- * @property {boolean} useNewQuestionRenderer
- */
+interface CourseOptions {
+  useNewQuestionRenderer: boolean;
+}
 
-/**
- * @typedef {Object} Tag
- * @property {string} name
- * @property {string} color
- * @property {string} [description]
- */
+interface Tag {
+  name: string;
+  color: string;
+  description?: string;
+}
 
-/**
- * @typedef {Object} Topic
- * @property {string} name
- * @property {string} color
- * @property {string} description
- */
+interface Topic {
+  name: string;
+  color: string;
+  description?: string;
+}
 
-/**
- * @typedef {Object} AssessmentSet
- * @property {string} abbreviation
- * @property {string} name
- * @property {string} heading
- * @property {string} color
- */
+interface AssessmentSet {
+  abbreviation: string;
+  name: string;
+  heading: string;
+  color: string;
+}
 
-/**
- * @typedef AssessmentModule
- * @property {string} name
- * @property {string} heading
- */
+interface AssessmentModule {
+  name: string;
+  heading: string;
+}
 
-/**
- * @typedef {Object} Course
- * @property {string} uuid
- * @property {string} name
- * @property {string} title
- * @property {string} path
- * @property {string} timezone
- * @property {boolean} exampleCourse
- * @property {CourseOptions} options
- * @property {Tag[]} tags
- * @property {Topic[]} topics
- * @property {AssessmentSet[]} assessmentSets
- * @property {AssessmentModule[]} assessmentModules
- */
+interface Course {
+  uuid: string;
+  name: string;
+  title: string;
+  path: string;
+  timezone: string;
+  exampleCourse: boolean;
+  options: CourseOptions;
+  tags: Tag[];
+  topics: Topic[];
+  assessmentSets: AssessmentSet[];
+  assessmentModules: AssessmentModule[];
+}
 
-/** @typedef {"Student" | "TA" | "Instructor" | "Superuser"} UserRole */
-/** @typedef {"UIUC" | "ZJUI" | "LTI" | "Any"} Institution */
+// This type is only used in legacy access rules. Any new rules are only defined for students.
+type UserRole = 'Student' | 'TA' | 'Instructor' | 'Superuser';
```

Comment:
As discussed on Slack, you said you were going to clean this up (as in remove it entirely). If you don't plan on doing that, let me know and I'll do it instead.

Reply:
What I removed was the institution. I can remove the user role as well, no problem.

Reply:
There are other places this is referenced; let me just open another PR to completely clean that up.

Reply:
Type removed in 47843119a7b5db20e326dda65d849ac64b034916. I replaced it with a string.